### ISSUE TITLE
Private is the default

### ISSUE THREAD

**Issue by leobenkel** [leobenkel/PSF-LoginServer#2]
Lots of fields are public when they could/should be private.

**Comment by Fate-JH** [comment]
If the class extends `Actor` then it doesn't matter because it's impossible to access these custom variables from the outside.  Every subclass of `Actor` is homogenized into just an `ActorRef` when it is interacted with.
Classic: https://doc.akka.io/api/akka/current/akka/actor/ActorRef.html
Typed: https://doc.akka.io/api/akka/current/akka/actor/typed/ActorRef.html
You can still change the scope on the variables but the only effect will be to make it look pretty.

**Comment by leobenkel** [comment]
yes but the compiler still needs to add those public functions to the registry which slows down compilation for no useful reasons.
